extensions/security/deployment/src/main/java/io/quarkus/security/deployment/SecurityConfig.java

```diff
@@ -34,4 +34,26 @@ public interface SecurityConfig {
      */
     @ConfigDocMapKey("provider-name")
     Map<String, String> securityProviderConfig();
+
+    /**
+     * If set to true, access to all methods of beans that have any security annotations on other members will be denied by
+     * default.
+     * E.g. if enabled, in the following bean, <code>methodB</code> will be denied.
+     *
+     * <pre>
+     *   &#064;ApplicationScoped
+     *   public class A {
+     *      &#064;RolesAllowed("admin")
+     *      public void methodA() {
+     *          ...
+     *      }
+     *      public void methodB() {
+     *          ...
+     *      }
+     *   }
+     * </pre>
+     */
+    @WithName("deny-unannotated-members")
+    @WithDefault("false")
+    boolean denyUnannotated();
```

Comment:
Why not name it `denyUnannotatedMembers()` and drop the `@WithName` annotation?

Reply:
I agree, good idea, thanks.

Reply:
working on other PR ATM, will change it later today